web/react/components/create_comment.jsx

```diff
@@ -105,14 +106,17 @@ module.exports = React.createClass({
             this.lastTime = t;
         }
     },
-    handleUserInput: function(messageText) {
+    handleUserInput: function(message) {
+        var messageText = CreatePost.truncateText(message);
```

Comment:
Fair enough, but then we really should come up with a solution to handle these kinds of functions as it's only going to get worse, especially as we begin to take a look at refactoring the duplicate code that already exists.